assets/js/hooks/editor_settings.js

```diff
@@ -17,6 +17,9 @@ const EditorSettings = {
     const editorAutoSignatureCheckbox = this.el.querySelector(
       `[name="editor_auto_signature"][value="true"]`,
     );
+    const editorCloseBracketsCheckbox = this.el.querySelector(
+      `[name="editor_close_brackets"][value="true"]`,
```

Comment:
Nitpick: I would rename it to "auto close brackets" for clarity. Other than that everything looks good :)

Reply:
Sure, I can do that. Funnily enough, I originally called it `editor_autoclose_delims` but renamed it after I saw what the setting was for the editor itself.